vllm/worker/habana_model_runner.py

```diff
@@ -488,6 +488,8 @@ class HabanaModelRunnerBase(ModelRunnerBase[TModelInputForHPU]):
     Helper class for shared methods between GPU model runners.
     """
     _model_input_cls: Type[TModelInputForHPU]
+    dummy_prompt_list: List[SequenceGroupMetadata]
```

Comment:
Can you please add it as a separate PR, as this does not relate to ray cluster initialization